spec/system/admin/reports/orders_and_distributors_spec.rb

```diff
@@ -34,27 +35,27 @@
       }
       let(:line_item1) {
         [completed_at, order.id, "John Doe", order.email, "123-456-7890", "Herndon",
-         "ABC", Spree::Product.first.name.to_s, "1g", "1", "none", "10.0", "none", "Check",
+         variant.sku, Spree::Product.first.name.to_s, "1g", "1", "none", "10.0", "none", "Check",
```

Comment:
It's better to use the actual value in specs. Otherwise we may miss a bug, for example if the sku is nil and the report fails to print the sku then this test would still pass.